[BUG] Victory Parade markers are rotated backwards by init script #12

cyrilepinat opened the pull request:
Fix issue https://github.com/HazelrahBL/EraOfTribesTTS/issues/12 :
- fix victory parade directions
- fix victory parade positions (since the board has been updated)

HazelrahBL commented:
Hey Cyril,

This looks right, but you also need to use Atom or another editor which uses the TTS plug-in API to store the code in the main `EoT_TTS.json` file. The file you've edited is just for reference, though it's the only way that you can actually see a change to the script in the history, so I think its very important to keep that reference there.

Take a look at cc33f9609719696a0ca1ce229901e21d21e5ae03 to see what I mean.

I really need to add instructions on how to contribute to this. I've got a few rules that I made for myself while making the updates and need to let people know what they are :)

cyrilepinat committed:
fix : [BUG] Victory Parade markers are rotated backwards by init script #12
- fix victory parade directions
- fix victory parade positions (after board update)
- update save file

HazelrahBL commented:
@cyrilepinat The changes are looking great!

Also, thank you for fixing some of the position changes. I swear I did that but the commit must have been lost in my mad dash to release.

Could you do me a favor though and squash the new commits into the first commit please? They really are "fixup" type commits and don't need to be in the history of the repository.

Here is a short guide if you've not done that before: https://linuxhint.com/how-to-squash-git-commits/

cyrilepinat commented:
You know you can do it at the moment where you merge the PR right ? :slightly_smiling_face: 
![image](https://user-images.githubusercontent.com/1631004/79348778-bda62d00-7f35-11ea-9aba-3f4d6a055dc1.png)

that's the way I use github but as you want, I can squash directly in my branch before you merge
